products/wso2_EI/business_process_orchestration/HumanTasksSamples/EmailNotificationSample/taskDeadlineWithNotifications/ClaimsApprovalTask.ht

```diff
@@ -0,0 +1,243 @@
+                    <![CDATA[
+                      <p>
+                        Approve this claim following corporate guideline
+                        <b>#4711.0815/7</b>
+                      </p>
+                      ]]>
+                </htd:description>
+                <htd:description xml:lang="de-DE" contentType="text/plain">
+                    Genehmigen Sie diese Schadensforderung entsprechend
+                    Richtlinie Nr. 4711.0815/7 ...
+                </htd:description>
+                <htd:description xml:lang="de-DE" contentType="text/html">
+                    <![CDATA[
+                      <p>
+                        Genehmigen Sie diese Schadensforderung entsprechend
+                        Richtlinie
+                        <b>Nr. 4711.0815/7</b>
+                        ...
+                      </p>
+                      ]]>
+                </htd:description>
+            </htd:presentationElements>
+
+            <htd:deadlines>
+
+                <htd:startDeadline name="sendNotifications">
+                    <htd:documentation xml:lang="en-US">
+                        if the claimed amount is less than 10000 - to the task's potential
+                        owners to remind them or their todo.
+					</htd:documentation>
+                    <htd:for>PT1M</htd:for>
+                    <htd:escalation name="reminder">
+
+						<htd:condition>
+				  				<![CDATA[htd:getInput("ClaimApprovalRequest")/test10:amount < 10000]]>	
+			 	 		</htd:condition>
+                        <htd:toParts>
+                            <htd:toPart name="firstname">
+                                htd:getInput("ClaimApprovalRequest")/test10:cust/test10:firstname
+                            </htd:toPart>
+                            <htd:toPart name="lastname">
+                                htd:getInput("ClaimApprovalRequest")/test10:cust/test10:lastname
+                            </htd:toPart>
+                        </htd:toParts>
+
+                        <htd:localNotification reference="tns:ClaimApprovalReminder">
+
+                            <htd:documentation xml:lang="en-US">
+                                Reuse the predefined notification "ClaimApprovalReminder".
+                                Overwrite the recipients with the task's potential owners.
+                            </htd:documentation>
+
+                        </htd:localNotification>
+
+                    </htd:escalation>
+                </htd:startDeadline>
+            </htd:deadlines>
+
+        </htd:task>
+    </htd:tasks>
+
+
+    <htd:notifications>
+        <htd:notification name="ClaimApprovalReminder">
+         <htd:renderings>
+         <htd:rendering type="wso2:email">
+<wso2:to name="to" type="xsd:string">user-email</wso2:to>
+<wso2:subject name="subject" type="xsd:string">Email Subject</wso2:subject>
+<wso2:body name="body" type="xsd:string">Hello $firstname$ $lastname$!</wso2:body>
```

Comment:
Shall we add the content type of the email to avoid unnecessary warnings? (<wso2:contentType name="contentType" type="xsd:string">text/plain</wso2:contentType>)

Reply:
Sure, Done changes.